content/admin/teamserver/config/integration/SSO.md

```diff
@@ -4,36 +4,32 @@ description: "TeamServer can be configured to utilize Single Sign On Authenticat
 tags: "SSO Single Sign-On configuration authentication"
 -->
 
-## Getting Started
+## Get Started
 
-Single Sign-On (SSO) is an authentication service that allows access to multiple applications using one set of credentials. Contrast can now be configured to use this service with a SAML 2.0 supported provider. 
+Single Sign On (SSO) is an authentication service that allows access to multiple applications using one set of credentials. You can configure Contrast to use this service with a SAML 2.0 supported provider. 
```

Comment:
FYI we use the hyphen in a lot of other places too. Docs, side nav link, UI, etc.